Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/TaniumThreatResponseV2.py

```diff
@@ -379,13 +381,21 @@ def alarm_to_incident(client, alarm):
 
     intel_doc = ''
     if intel_doc_id := alarm.get('intelDocId', ''):
-        raw_response = client.do_request('GET', f'/plugin/products/detect3/api/v1/intels/{intel_doc_id}')
-        intel_doc = raw_response.get('name')
-        alarm['intelDocDetails'] = raw_response
+        raw_response = client.do_request('GET', f'/plugin/products/'
+                                                f'{"threat-response" if client.api_version == "4.x" else "detect3"}'
```

Comment:
Since this is done all over the code and twice just here I suggest moving this check (`threat-response` vs `detect3`) to a property/function that'll be called anywhere that's relevant, e.g. class property - `threat_respons_endpoint` or a method (`get_threat_response_endpoint()`).